documentation/src/main/asciidoc/topics/con_xsite_concurrent_writes.adoc

```diff
@@ -0,0 +1,24 @@
+[id='conflicts-{context}']
+= Conflict Resolution with Cross-Site Replication
```

Comment:
this isn't implemented yet. :)

Reply:
no, these docs might be a bit ahead of what's currently available but I've got a separate PR for 10.x - I want to update docs in this PR for the irac work.